robotbase/src/main/java/frc/robot/Constants.java

```diff
@@ -98,6 +97,22 @@ public static final class ELEVATOR {
     public static final Distance[] ELEVATOR_HEIGHT_SETPOINTS = {};
   }
 
+  public static final class ALGAE_RUNNER {
+    public static final SparkBaseConfig MOTOR_CONFIG = new SparkMaxConfig()
+      .idleMode(IdleMode.kBrake)
+      .inverted(false);
+    
+    public static final double LEFT_MOTOR_P = 0;
+    public static final double LEFT_MOTOR_I = 0;
+    public static final double LEFT_MOTOR_D = 0;
+    public static final double LEFT_MOTOR_F = 0;
+
+    public static final ClosedLoopConfig CLOSED_LOOP_CONFIG_LEFT =
+      MOTOR_CONFIG.closedLoop
+        .pidf(LEFT_MOTOR_P, LEFT_MOTOR_I, LEFT_MOTOR_D, LEFT_MOTOR_F)
+        .outputRange(-1, 1);
```

Comment:
I don't believe we'll need closed loop for this subsystem so I think these constants can be removed (@NolanCampbell2357 could you confirm?)